tools/scene.py

```diff
@@ -125,23 +126,34 @@ def prettify(self, encoding=None, formatter='minimal', indent_width=4):
     if not os.path.exists(args.root):
         os.makedirs(args.root)
 
+    # Get the interactive tool path
+    interactive_dir = pathlib.Path(os.path.dirname(__file__)).parents
```

Comment:
Here is a bit involve a way to compute the root directory of the interactive-tool. Indeed, we need to take care of the use case where the user calls `python scene.py ...` directly from the `utils` directory.